ui/hooks/src/use-tags.ts

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
```

Comment:
I would rename this file to use-beams-by-tag, to better reflect what it actually does